Tests/ApodiniTests/OpenAPITests/OpenAPIConfigurationTests.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Created by Lorena Schlesinger on 28.01.21.
+//
+
+import XCTest
+@testable import Apodini
+
+final class OpenAPIConfigurationTests: ApodiniTests {
+    func testBuildDocumentWithConfiguration() {
+        let configuredOutputFormat: OpenAPIOutputFormat = .yaml
+        let configuredOutputEndpoint = "oas"
+        let configuredSwaggerUiEndpoint = "oas-ui"
+        let configuredTitle = "The great TestWebService - presented by Apodini"
+        
+        OpenAPIConfiguration(
+            outputFormat: configuredOutputFormat,
+            outputEndpoint: configuredOutputEndpoint,
+            swaggerUiEndpoint: configuredSwaggerUiEndpoint,
+            title: configuredTitle)
```

Comment:
Similar to my command above (very nitpicky)
```suggestion
            title: configuredTitle
        )
```

Reply:
👍 09f5109